tests/db/test_schema.py

```diff
@@ -125,9 +123,6 @@ def get_schema_update_command(dialect):
     return f"docker compose -f {docker_compose_yml} run --rm mlflow-{dialect} python {this_script}"
 
 
-@pytest.mark.skipif(
-    Version(sqlalchemy.__version__) > Version("1.4"), reason="Use 1.4 for schema check"
-)
```

Comment:
We used to test both sqlalchemy 1.x and 2.x. We chose to skip this test for 2.0, and dropped 1.4 later, and forgot to update this test.